ail_vru_uc1_scenario/Dockerfile

```diff
@@ -35,5 +35,4 @@ LABEL org.label-schema.build-date=${BUILD_DATE}
 
 ADD . /root/vehicle/config
 VOLUME /opt/carma/vehicle/config
-
-CMD  cp /root/vehicle/config/* /opt/carma/vehicle/config
+CMD cp -R /root/vehicle/config/. /opt/carma/vehicle/config
```

Comment:
Wait doesn't this break the config image. The previous command would copy all files in the `/root/vehicle/config/` directory but this one maybe only hidden files or am I reading that wrong. What is the difference between `../config/*` and `../config/.`?

Reply:
The main change was coming from two changes: `-R` recursive flag and `.` current directory. Combination of them lets us copy everything recursively which includes hidden files. However, since we didn't want to copy extra folders such as cdasim_config (although it is usually ignored by `.gitignore`), hooks, docs folders, recursive option was not ideal. So I opted to only copy the `.env` file specifically.